COLLADA2GLTF/COLLADA2GLTFWriter.cpp

```diff
@@ -95,7 +95,7 @@ namespace GLTF
         else
         {
             // We have the data in memory so use it
-            if (!root.loadDocument(asset->getInputFilePath(), fileData.data(), fileData.size())) {
+            if (!root.loadDocument(asset->getInputFilePath(), fileData.data(), (int)fileData.size())) {
```

Comment:
It's a `size_type` to `int`, which is 8 bytes to 4 bytes. I get a warning if I remove it. @lasalvavida did you build the 32-bit or 64-bit version of the converter?